src/content/guides/ecommerce-website.mdx

```diff
@@ -20,21 +22,86 @@ Centrapay will provide you with the following resources to allow you to start cr
 
 > We strongly recommend Centrapay APIs are invoked from your backend where your API key is securely stored.
 
-### Access the Centrapay SDK
-
-The Centrapay SDK enables acceptance of Centrapay payments on your website.
-It handles displaying the Centrapay button and launching the Centrapay checkout.
-
-Production: `https://sdk.centrapay.com/ecommerce/centrapay.js?merchantId={merchantId}`
+Centrapay will configure your business to accept test payments before you can accept live payments.
+Accepting live payments requires you to meet our certification requirements.
 
-### Centrapay Checkout Sample
+## Centrapay Checkout Sample
 
 A [sample Centrapay e-commerce application](https://github.com/centrapay/centrapay-checkout-sample) is available.
-It uses the Centrapay SDK on the frontend and Node.js on the backend.
+It includes demos of both the redirect method and the popup method.
 Please refer to the README.md file for configuration instructions.
 
+## Redirect Method
+
+The Redirect Method is the standard method used by most merchants.
+This method redirects the customer away from your website to Centrapay to complete the payment.
+The customer is redirected back to your website at the end of the process.
+
+### Overview
+
+``` mermaid
+sequenceDiagram
+autonumber
+  participant Consumer
+  participant MP as Merchant Site
+  participant MS as Merchant Server
+  participant CP as Centrapay
+
+  Consumer->>MP: Pay with Centrapay
+  MP->>+MS: Create Payment Request
+  MS->>+CP: Create Payment Request
+  CP-->>-MS: Payment Request created
+  MS-->>-MP: Payment Request created
+  MP->>CP: Redirect to Payment Request url
+  Note over CP: Show Payment Request details
+  Consumer->>CP: Complete payment
+  alt ✅ Success
+  CP->>MP: Redirect to redirectPaidUrl
+  else
+  CP->>MP: Redirect to redirectCancelUrl
+  end
+  MP->>+MS: Get Payment Request
+  MS->>+CP: Get Payment Request
+  CP-->>-MS: Return Payment Request
+  MS-->>-MP: Return Payment Request
+  opt Payment Request has status 'new'
+	MP->>+MS: Void Payment Request
+  MS->>+CP: Void Payment Request
+  CP-->>-MS: Return Payment Request
+  MS-->>-MP: Return Payment Request
+  end
+  MP-->>CP: Return
+  Note over MP: Checkout complete ✅
+```
+
+### Implementation
+
+> Your merchant must be configured with allowed domains for your redirect URLs.
```

Comment:
You have mentioned this twice

Reply:
@alexBinns yes I did because I thought people might not notice the first time 😂